C/services/south/south.cpp

```diff
@@ -432,12 +432,7 @@ void SouthService::start(string& coreAddress, unsigned short corePort)
 				else
 				{
 					Logger::getLogger()->debug("Plugin does not persist data");
-					try {
-						southPlugin->start();
-						started = true;
-					} catch (...) {
-						Logger::getLogger()->debug("Plugin start raised an exception");
-					}
+					started = true;
```

Comment:
This is fine for the non-async plugins, but how is plugin_start called if it is async?

Reply:
There is an else branch and around line 537

plugin->start() is called